uppsrc/CtrlCore/X11Ctrl.cpp

```diff
@@ -92,12 +93,12 @@ void GuiPlatformAfterMenuPopUp()
 	Ctrl::ProcessEvents();
 }
 
-void Ctrl::PaintCaret(SystemDraw& w)
-{
-	GuiLock __;
-	if(this == caretCtrl && WndCaretVisible)
-		w.DrawRect(caretx, carety, caretcx, caretcy, InvertColor);
-}
+//void Ctrl::PaintCaret(SystemDraw& w)  //aris002
```

Comment:
No need for annotation //aris002. You will be present in the git history, so this is usless and we will need to remove this comment anyway :)